models/Asset/Image/Thumbnail.php

```diff
@@ -273,6 +273,7 @@ public function getHtml($options = [])
             array_push($mediaConfigs, $thumbConfig->getItems()); //add the default config at the end - picturePolyfill v4
 
             foreach ($mediaConfigs as $mediaQuery => $config) {
+                $thumbConfig->setItems($config);
```

Comment:
I would be careful not to override the default config. It was cloned and used until this PR #11632 so optimal change would be to use the cloned object like [here](https://github.com/pimcore/pimcore/pull/11632/files#diff-c1144c6a30525acb9f7006167ec8142edec7f4e974123db8824f11af57cfb4e6L226-L227)

Reply:
ahh, thank you for the heads-up! 
Did i got it right on https://github.com/pimcore/pimcore/pull/15253/commits/f8f144d5bfe9ec7178fc61640698bc518d9ada11 ?

Reply:
could you please try cloning outside loop and selecting media inside `$thumbConfigRes->selectMedia($mediaQuery);`?